### PR TITLE
Submitter celltype heatmap

### LINKED ISSUE #37: Supplemental cell type figures
Breaking this out of #7, we will want a supplemental figure for cell type annotation that includes: 

- Example diagnostic plots for CellAssign and SingleR
- Comparison of submitter annotations to SingleR and CellAssign
- Some TBD information on how we decided to use SingleR and CellAssign and the references.

### PR DESCRIPTION
**Pull request by allyhawkins**
Closes #37 

This adds the next supplemental cell type plot, the heatmap comparing submitter cell types to annotations with SingleR and CellAssign. I tried to pick a library that had cell assignments in both SingleR and CellAssign and wasn't just assigning everything to Unknown for CellAssign. There's definitely still some Unknown, which is very much to be expected, but they line up with tumor cells, which I think is good. 

Again, I formatted the plot in the same way we create the plot in the supplemental report and also how we create the other heatmap we added in #38. We will also want to decide about legend placement, but I think we can do that when we combine everything into one figure. 

Tagging @jaclyn-taroni for aesthetic comments.

### CODE REVIEW

File: scripts/FigS4C_submitter-celltypes-heatmap.R [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# This script is used to generate the heatmap comparing submitter annotations to SingleR/ CellAssign
```
Comment (sjspielman) [inline scripts/FigS4C_submitter-celltypes-heatmap.R:1]:
```suggestion
# This script is used to generate the heatmap comparing submitter annotations to SingleR and CellAssign
```

**Review by sjspielman — APPROVED** [review]
LGTM, no real notes! 🐱

File: figures/pngs/FigS4C_submitter-heatmap.png
Comment (jaclyn-taroni) [inline figures/pngs/FigS4C_submitter-heatmap.png:1]: 👍🏻

**Comment by allyhawkins** [comment]
@sjspielman Note that I added a description of this figure and script to the README, if you want to take a look at https://github.com/AlexsLemonade/scpca-paper-figures/pull/40/commits/47105643f4bac564e7d3e7fecb29779a617b8ae7 before I merge.

**Comment by sjspielman** [comment]
> Note that I added a description of this figure and script to the README

Bad reviewer (me)! Looks good!

**Pull request merged**